executor/batch_point_get.go

```diff
@@ -359,8 +361,7 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 	e.values = make([][]byte, 0, len(values))
 	for i, key := range keys {
 		val := values[string(key)]
-		// track memory
-		e.memTracker.Consume(int64(len(key)) + int64(len(val)))
+		e.memTracker.Consume(int64(cap(val)))
```

Comment:
Ditto, not all `val` will be append to `e.values`. I think it would be better to put after `e.values = append(e.values, val)`.